RealmSwift/Tests/ObjectSchemaTests.swift

```diff
@@ -24,7 +24,9 @@ class ObjectSchemaTests: TestCase {
 
     override func setUp() {
         super.setUp()
-        objectSchema = Realm().schema["SwiftObject"]
+        autoreleasepool {
+            self.objectSchema = Realm().schema["SwiftObject"]
```

Comment:
Do we even need this as a member? As said in the past this pattern has caused issues and should be avoided if possible.